tasks/config/routes.rb

```diff
@@ -6,7 +6,10 @@
   post 'tasks/' => 'tasks#create'
   delete 'tasks/:id' => 'tasks#destroy'
   get 'tasks/new' => 'tasks#new'
+  get 'tasks/:id/edit' => 'tasks#edit'
   get 'tasks/:id' => 'tasks#show'
+  patch 'tasks/:id' => 'tasks#update', :as => :task
```

Comment:
We'll be talking about "named routes" (what you get when using `as: :name`) later this week. For now however, it's probably best to avoid using them.